src/Modera/SecurityBundle/DependencyInjection/Configuration.php

```diff
@@ -78,7 +78,16 @@ public function getConfigTreeBuilder()
                         ->scalarNode('number_required')
                             ->defaultFalse()
                         ->end()
-                        ->scalarNode('capital_letter_required')
+                        ->scalarNode('letter_required')
+                            // capital_or_non_capital, capital_and_non_capital, capital, non_capital
```

Comment:
Introduce constants to PasswordConfigInterface and perform validation right here if the given value is correct.